src/core/Akka.TestKit/TestKitBase_Expect.cs

```diff
@@ -40,18 +40,16 @@ public abstract partial class TestKitBase
             string hint = null,
             CancellationToken cancellationToken = default)
         {
-            return ExpectMsgAsync<T>(duration, hint, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+            return ExpectMsgAsync<T>(duration, hint, cancellationToken).Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.